README.md

```diff
@@ -97,137 +55,98 @@ After you're finished please remove all the comments and instructions!
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **Implement Authenticated endpoint**
+- **Implement CRUD functionality for hotels model**
+- **Implement CRUD functionality for rooms model**
+- **Implement CRUD functionality for booking model**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
+## 🚀 LIVE DEMO <a name="live-demo"></a>
+- [Link to Live Demo](http://hotel-backend.fly.dev/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FRONT END LINK -->
 
-## 🚀 Live Demo <a name="live-demo"></a>
+## 🚀 Frontend Link <a name="frontend-link"></a>
 
-> Add a link to your deployed project.
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Link to Frontend Code](https://github.com/t-bello7/hotel-frontend)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
 To get a local copy up and running, follow these steps.
 
 ### Prerequisites
 
 In order to run this project you need:
-
-<!--
-Example command:
-
-```sh
- gem install rails
-```
- -->
+- Ruby 
+- Postgres
 
 ### Setup
 
 Clone this repository to your desired folder:
 
-<!--
-Example commands:
-
 ```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
+  git clone https://github.com/t-bello7/hotel-backend.git
 ```
---->
 
 ### Install
 
 Install this project with:
 
-<!--
-Example command:
-
 ```sh
-  cd my-project
-  gem install
+  cd hotel-backend
+  bundle install
 ```
---->
 
 ### Usage
 
 To run the project, execute the following command:
 
-<!--
-Example command:
-
 ```sh
-  rails server
+  bin/rails server
 ```
---->
 
+```sh
+  bin/rails db:create 
+  bin/rails db:migrate
+```
 ### Run tests
 
 To run tests, run the following command:
 
-<!--
-Example command:
-
 ```sh
-  bin/rails test test/models/article_test.rb
+  rspec spec 
```

Comment:
- [ ]  When I run `rspec spec` according to your instructions, I get this:

![image](https://user-images.githubusercontent.com/57421823/216636137-5dac6078-b7e7-4cd5-bd2d-7425b652bda3.png)

If these tests are passing on your end and there are special procedures to run your tests properly without failing, kindly add those instructions to your readme.  👍